### ISSUE TITLE
from torch.nn.modules.batchnorm import _BatchNorm

### ISSUE THREAD

**Issue by Guapicat0** [open-mmlab/mmsegmentation#1999]
how to solve the question about"Cannot find reference '_BatchNorm' in 'batchnorm.pyi' '

**Comment by MeowZheng** [comment]
Could you provide more information following the issue template?

**Comment by Guapicat0** [comment]
i&nbsp; want tu run the code of Mobilenet_v2.py,
pycharm reminds me an error.
like photo indicates





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "open-mmlab/mmsegmentation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月1日(星期四) 下午3:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [open-mmlab/mmsegmentation] from torch.nn.modules.batchnorm import _BatchNorm (Issue #1999)





 
Could you provide more information following the issue template?

**Comment by Guapicat0** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "open-mmlab/mmsegmentation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月1日(星期四) 下午3:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [open-mmlab/mmsegmentation] from torch.nn.modules.batchnorm import _BatchNorm (Issue #1999)





 
Could you provide more information following the issue template?
